api/envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.proto

```diff
@@ -29,9 +29,8 @@ message GcpAuthnFilterConfig {
   config.core.v3.RetryPolicy retry_policy = 2;
 }
 
+// Audience is the URL of the receiving service that performs token authentication.
+// It will be provided to the filter through cluster's typed_filter_metadata.
 message Audience {
-  // The map of audience key to audience value.
-  // The key is defined as the contract with control plane in the configuration. It is fixed string "audience_key".
-  // The value is URL of the receiving service that performs token authentication.
-  map<string, string> audience_map = 1;
```

Comment:
Changing the type of this field is a breaking API change.

https://github.com/envoyproxy/envoy/blob/main/api/API_VERSIONING.md#backwards-compatibility

Reply:
Thanks for the review! I have thought about backwards-compatibility.

Can this be an exception since actually no one use it yet? It is a pretty new feature that is only introduced a few days ago.

Reply:
Ah, okay, I didn't realize that it was added recently.  The API versioning policy does provide an exception for that case:

> Changes made within 14 days of the introduction of a new API field or message, provided the new field or message has not been included in an Envoy release.

So in that case, this seems fine to me.